src/components/tabs/Tabs.tsx

```diff
@@ -36,10 +36,24 @@ type Props = {
 	sharedStore?: string;
 };
 
+const tabOrdering = {
+	npm: 0,
+	pnpm: 1,
+	yarn: 2,
+	bun: 3,
+};
+
 export default function Tabs({ sharedStore, ...slots }: Props) {
 	const tabId = genTabId();
-	const tabs = Object.entries(slots).filter(isTabSlotEntry);
-	const panels = Object.entries(slots).filter(isPanelSlotEntry);
+	// npm then pnpm then yarn then bun
+	const sortedSlots = Object.entries(slots).sort(([a], [b]) => {
```

Comment:
Before this, the tabs were getting sorted alphabetically, so Bun was first. This sets a fixed order for the tabs based on how the current appear in the docs.